app/models/User.coffee

```diff
@@ -619,6 +619,7 @@ module.exports = class User extends CocoModel
   hideDiplomatModal: -> features?.china ? false
   showChinaRemindToast: -> features?.china ? false
   showOpenResourceLink: -> not (features?.china ? false)
+  shouldUpsell: -> (not (features?.china ? false)) and (@get('preferredLanguage') isnt 'nl-BE')
```

Comment:
Why are we checking for the Dutch language code here?

Reply:
That's the old logic?

Reply:
old logic. You can find it at EnrollmentView#line116 in old file

Reply:
Ah of course, I missed that!